modules/wsl-distro.nix

```diff
@@ -47,12 +61,22 @@ with builtins; with lib;
       };
 
       # WSL is closer to a container than anything else
-      boot.isContainer = true;
+      boot = {
+        isContainer = true;
+
+        binfmt.registrations = mkIf cfg.interop.enable {
```

Comment:
The wslConf options are in a weird spot currently - `interop` does the binfmt registration in the host namespace, which mostly leaks into the systemd namespace by accident and I'm actually intentionally keeping `appendWindowsPath` so `WSLPATH` is still set.

Reply:
At least for me, the binfmt registration from the host namespace works just fine, so overwriting it may not be neccessary. Having `WSLPATH` set, so that the user can append it themself later is a good idea - I hadn't thought about that

Reply:
The host registration works iff systemd-binfmt is not used, if you enable systemd-binfmt, it removes all existing registrations.

Reply:
I see. Then this has to stay as it is